src/modules/todo.js

```diff
@@ -0,0 +1,103 @@
+const taskContainer = document.querySelector('.task-container');
+
+export default class myTodo {
+  constructor() {
+    this.tasks = [];
+  }
+
+    addTask = (description) => {
+      const myTask = {
+        description,
+        completed: false,
+        index: this.tasks.length + 1,
+      };
+      this.tasks.push(myTask);
+      this.setStorage();
+    }
+
+    remove = (id) => {
+      this.resetIndex();
+      this.tasks.forEach((task) => {
+        if (task.index === Number(id)) {
+          this.tasks
+            .splice(task.index - 1, 1);
+        }
+      });
+    }
+
```

Comment:
- [ ] As mentioned by the previous reviewer, when you delete a task, the remaining tasks' indexes should be updated in order to maintain uniqueness and also the order. In your application, when I delete a task, other task indexes are not updated, so please correct that 😃 

![image](https://user-images.githubusercontent.com/90222110/184380759-8fc1d425-1181-401f-abe8-380bb051e7d7.png)